src/model/migrations.js

```diff
@@ -511,13 +509,9 @@ export default async function runMigrations() {
         const { address } = wallet.addresses[j];
         const hiddenCoins = await getHiddenCoins(address, network);
         const pinnedCoins = await getPinnedCoins(address, network);
-        const savingsToggle = await getSavingsToggle(address, network);
```

Comment:
I've confirmed with Christian that it is okay to not "migrate" these the same way we do for hidden/pinned coins. (It's okay if the open states go back to their default states just this once.) This allows us to remove the localstorage functions and I have updated the migration file.